### DISCUSSION TITLE
Koillection and phpBB

### DISCUSSION THREAD

**Discussion by micavero** [benjaminjonard/koillection#294] [Ideas]
Hello, 
I took great interest in you software. 
I'm managing a free website/forum of plant collectors based on phpBB and i was searching for a way to let my user manage/show what plants they grow. Is that possible with Koillection? I see that with Koillection you can have several users on the same instance; but we would also need a shared "plant database" shared among all the user of the koillection website, with unique identifier for each specie/clone in order to be able to unambiguosly identify them and **exchange/trade** them (wishlist could be very useful in that regard). The plant database should be also manageable by admins because some users might insert in their collection plants wich are patented and shouldn't be exchangeable/showable, while the other stuff could/should be publicly available in order to be browsable by people not registered on the site. Is that doable with koillection getting the user list from phpBB? Is there any way to bridge koillection with any forum software?

**Comment by benjaminjonard** [comment]
Hello, thank you for in interest in Koillection.

Unfortunately I don't think it could provide the features you are looking for. 
I created this project so that every user is absolutely free to manage their collection as they want, without restriction. There is no global database where users can choose items from to add to their collection, every user adds its own items. It's designed to be a personal collection manager, not really a collaborative one.
Right now there is no bridge between Koillection and any kind of forums.

Sorry to disappoint you, hope you'll find what you seek :slightly_smiling_face:
